src/main/java/com/google/finapp/SpannerDaoInterface.java

```diff
@@ -32,8 +30,10 @@
   /**
    * Inserts a new row to the Account table in the database.
    *
+   * @param accountId unique id of account
```

Comment:
some of these javadocs might be unnecessary? `accountId` and `balance` are self-explantory. just a suggestion to simplify the javadocs.

Reply:
`accountId` can be removed, but `balance` description contains the non-negative precondition